src/YLayoutBox.h

```diff
@@ -31,7 +31,9 @@
 
 class YLayoutBoxPrivate;
 
-
+/**
+ * A vertical or horizontal stacking of widgets, implementing HBox and VBox.
```

Comment:
a bit misleading to talk about its usage in different project as HBox and VBox. I think note about H/VBox is not needed.